.github/workflows/actionlint.yml

```diff
@@ -0,0 +1,81 @@
+# This file is synced from the `.github` repository, do not modify it directly.
+name: Workflow Syntax
```

Comment:
Can this have Actionlint in the name so it better matches the filename?

Reply:
#200